city_scrapers/spiders/chi_state_university.py

```diff
@@ -0,0 +1,151 @@
+                        start=self._parse_start(item),
+                        end=None,
+                        all_day=False,
+                        time_notes=self._parse_time_notes(item),
+                        location=self._parse_location(item),
+                        links=self._parse_links(item),
+                        source=self._parse_source(response),
+                    )
+                    meeting["status"] = self._get_status(meeting)
+                    meeting["id"] = self._get_id(meeting)
+
+                    yield meeting
+
+    def _parse_description(self, item):
+        parts = item.xpath(".//text()").extract()
+        description = "".join([str(x) for x in parts])
+        description = (
+            description.replace("\xa0", " ")
+            .replace("\n", " ")
+            .replace("                                     ", " ")
+            .replace("\t", " ")
+            .strip()
+        )
+        return description
+
+    def _parse_classification(self, i):
+        classifications = [BOARD, BOARD, COMMITTEE]
+        return classifications[i]
+
+    def _parse_start(self, item):
+        text = item.xpath(".//text()").extract()[0]
+        text = text.replace("\xa0", "").strip()
+
+        today = datetime.now()
+        yearMatch = today.year
+        monthMatch = today.month
+        dayMatch = today.day
+        hourMatch = 0
+        minuteMatch = 0
+
+        try:
+            yearMatch = int(re.search(r"\d{4}", text).group(0))
+        except AttributeError:
+            pass
+
+        try:
+            dayMatch = int(re.search(r"\d{1,2}", text).group(0))
+        except AttributeError:
+            pass
+
+        try:
+            months = {m.lower() for m in month_name[1:]}
+            monthMatch = next(
+                (word for word in text.split() if word.lower() in months), None
+            )
+            monthMatch = datetime.strptime(monthMatch, "%B").month
+        except AttributeError:
+            pass
+
+        try:
+            minuteMatch = re.search(r":([0-5][0-9])", text).group(0)
+            minuteMatch = int(minuteMatch.replace(":", ""))
+        except AttributeError:
+            pass
+
+        try:
+            hourMatch = re.search(r"(1[0-2]|0?[1-9]):", text).group(0)
+            hourMatch = int(hourMatch.replace(":", ""))
+        except AttributeError:
+            pass
+
+        if "p.m." in text:
+            hourMatch += 12
+
+        return datetime(yearMatch, monthMatch, dayMatch, hourMatch, minuteMatch)
+
+    def _parse_time_notes(self, item):
+        text = " ".join(item.xpath(".//text()").extract())
+        text = text.replace("\xa0", "").strip()
+
+        notes = ""
+        if "reschedule" in text.lower() or "postpone" in text.lower():
+            notes = text
+        if "tbd" in text.lower():
+            notes = "Meeting time TBD"
+        return notes
+
+    def _parse_location(self, item):
```

Comment:
It's fine to return a static location, but we'll want to add a `_validate_location` method to make sure that it hasn't changed. You can find an example here https://github.com/City-Bureau/city-scrapers/blob/c6771d58209fa9ade4aa3768fa3ddb23b45c09fc/city_scrapers/spiders/chi_ssa_34.py#L61-L63